composer.json

```diff
@@ -24,6 +24,7 @@
         "php": ">=5.4.8",
         "symfony/event-dispatcher": "~2.3",
         "symfony/options-resolver": "~2.3",
+        "symfony/filesystem": "~2.3",
```

Comment:
Let’s not reinvent the wheel and just use symfony/filesystem. The only problem I have with this is that it is an *optional* dependency. @dbu Should we move symfony/filesystem to `require-dev` and add a note in the docs that users should install the dependency when they want to use tagging in Symfony Cache?